ui/app/components/mount-backend/type-form.hbs

```diff
@@ -4,39 +4,30 @@
 ~}}
 
 {{#each (array "generic" "cloud" "infra") as |category|}}
-  <h3 class="title box-radio-header">
+  <Hds::Text::Display class="has-top-padding-m has-bottom-padding-s" @tag="h2" size="400">
     {{capitalize category}}
-  </h3>
-  <div class="box-radio-container">
```

Comment:
Same here, I think we can delete the `box-radio-container` class from `box-radio.scss`

Reply:
Yep - just haven't pushed the commit where I delete all of Box radio related stuff yet :)